src/client/gui/lib/settings/virtualization_settings.dart

```diff
@@ -28,36 +28,27 @@ class VirtualizationSettings extends ConsumerWidget {
         label: 'Driver',
         width: 260,
         value: driver,
-        items: drivers,
+        items: mpPlatform.drivers,
         onChanged: (value) {
           if (value == driver) return;
-          ref.read(driverProvider.notifier).set(value!);
+          ref
+              .read(driverProvider.notifier)
+              .set(value!)
+              .onError(ref.notifyError((e) => 'Failed to set driver: $e'));
         },
       ),
       const SizedBox(height: 20),
       if (networks.isNotEmpty)
         Dropdown(
```

Comment:
Another nitpick: with the default window size, the bridged network dropdown is cut off the screen unless one scrolls. Anything that can be done to save 15 pixels or so and let everything fit?

Reply:
We could either make the window larger or just decrease the spacing between the elements.